flask_security/forms.py

```diff
@@ -22,7 +22,7 @@
 from .confirmable import requires_confirmation
 from .utils import _, _datastore, config_value, get_message, \
     localize_callback, url_for_security, validate_redirect_url, \
-    verify_and_update_password
+    verify_and_update_password, encrypt_password
```

Comment:
@sh4r3m4n can you please keep the imports sorted alphabetically? It's not strictly checked, but it would be nice to keep the imports organized. Thanks!